tdrs-backend/requirements.txt

```diff
@@ -1,53 +0,0 @@
--i https://pypi.org/simple
```

Comment:
Dependabot can manage Python dependencies using the `Pipfile` so we no longer need to maintain this file. (It's out of date anyway - note this [closed vulnerability](https://github.com/raft-tech/TANF-app/security/dependabot/tdrs-backend/requirements.txt/cryptography/closed), in our Pipfile.lock version 3.3.2 of cryptography is currently in use)